README.md

```diff
@@ -536,6 +536,7 @@ There are a few variables that are intended to be customized via `.dir-locals.el
 
 They're all set to `nil` by default, but by setting them you'll override the
 default commands per each supported project type.
+These variables can be strings to run external commands or elisp functions.  
```

Comment:
elisp -> Emacs Lisp

Perhaps we should also have 1-2 examples here for clarity's sake.